src/_reducers/TickReducer.js

```diff
@@ -18,22 +28,24 @@ export default (state = initialState, action) => {
                 epoch: +tick.epoch,
                 quote: +tick.quote,
             };
-            return state.update(symbol, List.of(), v => v.takeLast(60).push(newTick));
+            return state.update(symbol, List.of(), v => v.takeLast(4999).push(newTick));
```

Comment:
Do we really want to get the maximum amount of data each time?
I am not sure. This is probably hundreds of kbs of data.